### PR TITLE
Fix postinstall page test

### PR DESCRIPTION
**Pull request by sballesteros**
**Reviewer:**

## Description:

Due to a bug the current test for the post-install page always passes. This PR solves that by fixing the async/await logic.

## Steps to test this PR:
- Run the first test in `integration-test/background/atb.js`
- [ ] check that it passes

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

File: integration-test/background/atb.js [hunk 1]
```diff
@@ -25,12 +25,9 @@ describe('install workflow', () => {
             // wait for post install page to open
             // if it never does, jasmine timeout will kick in
             while (!postInstallOpened) {
+                const urls = await Promise.all(browser.targets().map(target => target.url()))
```
Comment (sballesteros) [inline integration-test/background/atb.js:28]: Note: `browser.targets()` returns an array (_not_ a promise) and `target.url()` return a promise (see [docs](https://pptr.dev/#?product=Puppeteer&version=v7.1.0&show=api-browsertarget))

**Review by jonathanKingston — APPROVED** [review]
LGTM

**Comment by sballesteros** [comment]
Thanks for the review @jonathanKingston. Quick follow-up question as I am not familiar with the process here. Should I go ahead and merge this PR or is that done differently?

**Comment by jonathanKingston** [comment]
@sballesteros the process is outlined here: https://app.asana.com/0/312629933896096/312783424879089 (I'm pretty new here too though) create a task there but feel free to merge it once you've done that and put it in the right section.

**Pull request merged**